src/org/opendatakit/briefcase/reused/http/CommonsHttp.java

```diff
@@ -147,6 +151,16 @@ public void unsetProxy() {
       throw new HttpException("The connection has timed out", e);
     } catch (IOException e) {
       throw new UncheckedIOException(e);
+    } finally {
+      try {
```

Comment:
I wish we could use try with resources but I guess it's up to the HTTP client's implementation. I recon that this would work as we want.

Reply:
I agree. There's a chance that the code can be refactored at a later point to possibly accomplish that.